docs/src/pages/[platform]/connected-components/authenticator/headless/headless-usage.web.mdx

```diff
@@ -86,7 +86,7 @@ Here's an example that uses the `toResetPassword` trigger transition, to create
         <TableRow>
           <ResponsiveTableCell label="Name">`codeDeliveryDetail`</ResponsiveTableCell>
           <ResponsiveTableCell label="Description">Provides detail on where confirm sign up code is sent to.</ResponsiveTableCell>
-          <ResponsiveTableCell label="Type">[`CodeDeliveryDetail`](https://github.com/aws-amplify/amplify-js/blob/6c4d4b5fc737652f88dcdfa26bdb02a1defdfa2e/packages/amazon-cognito-identity-js/index.d.ts#L9-L13)</ResponsiveTableCell>
```

Comment:
Removed this link as it points to the Amplify JS repo, which we do not maintain and could potentially change without us being aware